### PR TITLE
Add cffi to runtime requirements

### PR DESCRIPTION
**Pull request by maxnoe**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by maxnoe** [comment]
Using `pip check` complained that `zstandard` is missing the `cffi` dependency after installing from conda forge, this should fix it.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -24,6 +24,7 @@ requirements:
     - cffi >=1.8
   run:
     - python
+    - cffi >=1.8
 
 test:
```
Comment (xhochy) [inline recipe/meta.yaml:29]:
```suggestion
test:
    requires:
      - pip
    commands:
      - pip check
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -24,8 +24,13 @@ requirements:
...
+    requires:
+      - pip
+    commands:
+      - pip check
```
Comment (xhochy) [inline recipe/meta.yaml:33]:
```suggestion
  requires:
    - pip
  commands:
    - pip check
```

**Comment by rmax** [comment]
I could reproduce it:
```
(base) root@16b5f0f237b7:/# conda create -n testenv python=3.8 zstandard
Collecting package metadata (current_repodata.json): done
Solving environment: done
Segmentation fault
(base) root@16b5f0f237b7:/# conda create -n testenv2 python=3.8 zstandard
Collecting package metadata (current_repodata.json): done
Solving environment: done

## Package Plan ##

  environment location: /opt/conda/envs/testenv2

  added / updated specs:
    - python=3.8
    - zstandard


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    python-3.8.6               |h852b56e_0_cpython        22.1 MB  conda-forge
    zstandard-0.14.0           |   py38h950e882_2         582 KB  conda-forge
    ------------------------------------------------------------
                                           Total:        22.7 MB

The following NEW packages will be INSTALLED:

  _libgcc_mutex      conda-forge/linux-64::_libgcc_mutex-0.1-conda_forge
  _openmp_mutex      conda-forge/linux-64::_openmp_mutex-4.5-1_gnu
  ca-certificates    conda-forge/linux-64::ca-certificates-2020.6.20-hecda079_0
  certifi            conda-forge/linux-64::certifi-2020.6.20-py38h924ce5b_2
  ld_impl_linux-64   conda-forge/linux-64::ld_impl_linux-64-2.35-h769bd43_9
  libffi             conda-forge/linux-64::libffi-3.2.1-he1b5a44_1007
  libgcc-ng          conda-forge/linux-64::libgcc-ng-9.3.0-h5dbcf3e_17
  libgomp            conda-forge/linux-64::libgomp-9.3.0-h5dbcf3e_17
  libstdcxx-ng       conda-forge/linux-64::libstdcxx-ng-9.3.0-h2ae2ef3_17
  ncurses            conda-forge/linux-64::ncurses-6.2-he1b5a44_2
  openssl            conda-forge/linux-64::openssl-1.1.1h-h516909a_0
  pip                conda-forge/noarch::pip-20.2.4-py_0
  python             conda-forge/linux-64::python-3.8.6-h852b56e_0_cpython
  python_abi         conda-forge/linux-64::python_abi-3.8-1_cp38
  readline           conda-forge/linux-64::readline-8.0-he28a2e2_2
  setuptools         conda-forge/linux-64::setuptools-49.6.0-py38h924ce5b_2
  sqlite             conda-forge/linux-64::sqlite-3.33.0-h4cf870e_1
  tk                 conda-forge/linux-64::tk-8.6.10-hed695b0_1
  wheel              conda-forge/noarch::wheel-0.35.1-pyh9f0ad1d_0
  xz                 conda-forge/linux-64::xz-5.2.5-h516909a_1
  zlib               conda-forge/linux-64::zlib-1.2.11-h516909a_1010
  zstandard          conda-forge/linux-64::zstandard-0.14.0-py38h950e882_2


Proceed ([y]/n)? y


Downloading and Extracting Packages
python-3.8.6         | 22.1 MB   | ####################################################################################################################################### | 100%
zstandard-0.14.0     | 582 KB    | ####################################################################################################################################### | 100%
Preparing transaction: done
Verifying transaction: done
Executing transaction: done
#
# To activate this environment, use
#
#     $ conda activate testenv2
#
# To deactivate an active environment, use
#
#     $ conda deactivate

(base) root@16b5f0f237b7:/# conda activate testenv2
(testenv2) root@16b5f0f237b7:/# pip check
zstandard 0.14.0 requires cffi, which is not installed.
```

**Pull request merged**